docs/ref/models/fields.txt

```diff
@@ -1424,7 +1424,8 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
 .. attribute:: ForeignKey.to_field
 
     The field on the related object that the relation is to. By default, Django
-    uses the primary key of the related object.
+    uses the primary key of the related object, therefore ensure the referenced
```

Comment:
I would probably phrase it:

`uses the primary key of the related object, therefore ensuring the referenced field...